consensus/consensus_types/src/lib.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+#[macro_use]
+extern crate failure;
```

Comment:
Don't use extern crate and instead prefer bringing macros into scope using `use`